plugins/fabrik_element/fileupload/fileupload.php

```diff
@@ -1066,8 +1062,8 @@ protected function _getAllowedExtension()
 
 		if ($allowedFiles != '')
 		{
-			// $$$ hugh - strip spaces, as folk often do ".foo, .bar"
-			preg_replace('#\s+#', '', $allowedFiles);
+			// $$$ hugh - strip spaces and leading ., as folk often do ".bmp, .jpg"
+			preg_replace('#(\s+|^)\.?#', '', trim($allowedFiles));
```

Comment:
I agree - this is still not quite correct.

We want to strip:

a. White space;
b. Leading . at the beginning of the string;
c. the . immediately after a ,